### PR TITLE
Change to src layout and release 1.0.3

### PR DESCRIPTION
**Pull request by naruxde**
Reorder project to src layout and add some pythonize files for build and deploy process.

Makefile can create a virtual environment
Makefile can build source and wheel distribution
Makefile can install package to test it in the venv

Version number for entire project in __about__.py only (setup.py uses this one).

Parameter --version always outputs the version of the software on the CLI.

Long description is generated from README.md (also for PyPi)

### CODE REVIEW

**Comment by naruxde** [comment]
Tested with [hat-files.zip](https://github.com/RevolutionPi/python3-revpi-device-info/files/10294892/hat-files.zip)

File: src/revpi_device_info/cli.py [hunk 1]
```diff
@@ -54,46 +54,53 @@ def output_text(device_info: RevPiDeviceInfo, line_length: int = 60):
     print(f"Format Version:\t{device_info.format_version}")
 
 
-def main() -> None:
+def main() -> int:
     parser = argparse.ArgumentParser(
-        description="Human readable RevPi Device Info")
+        description="Human readable RevPi Device Info",
```
Comment (iluminat23) [inline src/revpi_device_info/cli.py:59]: This is some kind of white space change. This is from your IDE? seperate commit?

Comment (naruxde) [reply src/revpi_device_info/cli.py:59]: This has been changed on purpose. This results in a cleaner diff when adding parameters and improves code readability.

Comment (iluminat23) [reply src/revpi_device_info/cli.py:59]: Please do these changes in an extra commit. The best would be as first commit.

Comment (naruxde) [reply src/revpi_device_info/cli.py:59]: I would have done that for a larger project. This project is very new and tiny, the change file has less than 100 lines of source code. I think it's okay the way it is in this case.

Comment (iluminat23) [reply src/revpi_device_info/cli.py:59]: So splitting the commits should not be much of a burden.

**Comment by naruxde** [comment]
Didn't get that with the extra cleanup commit. Everything was a mess at the end. Check it out again if we can't do it like this...

**Pull request closed without merging**